server/views/components/customDisplay/crmEvidence.njk

```diff
@@ -1,17 +1,25 @@
-{% from "../formField.njk" import formField %}
+{% from "../formTable.njk" import formTable %}
 {% from "../formSubsection.njk" import formSubsection %}
 
 {% macro crmEvidence(data, crmType) %}
 
-    {% set fields = [] %}
+    {% set tableEvidence = {
+        head: [],
+        rows: []
+    } %}
 
     {% for item in data %}
         {#  prefix fileKey with CRM14/ path if fileKey starts with "att_" #}
         {% set fileKey = "CRM14/" + item.key if crmType === "CRM 14" and (item.key | truncate(4, true, '')) === "att_" else item.key %}
         {% set linkUrl = "/download-evidence?fileKey=" + (fileKey | urlencode) + "&fileName=" + (item.name | urlencode) %}
-        {% set list = fields.push({ label: item.name, value: { href: linkUrl, text: "download" } , type: 'link' }) %}
```

Comment:
As type: 'link' is no longer used. Suggested clean up of the following code:-

Remove from the following formField.njk
```
    {% elif field.type === 'link' %}
        <a href="{{ field.value.href }}" class="govuk-link govuk-link--no-visited-state">{{ field.value.text }}</a>
```

Remove 'link' from FieldType in crmDisplay/index.ts
      `  type FieldType = 'currency' | 'date' | 'time' | 'percent' | 'link' | 'timeAndCost' | 'totalAndCost'`